tests/plugins/UnitTests/FindLargeFiles.Tests/FindLargeFilesFormTests.cs

```diff
@@ -0,0 +1,28 @@
+﻿using GitExtensions.Plugins.FindLargeFiles;
+
+namespace DeleteUnusedBranchesTests
+{
+    [TestFixture]
+    public class FindLargeFilesFormTests
+    {
+        [Test]
+        public async Task GenerateCommand_without_deletions_should_return_expected()
+        {
+            await Verifier.Verify(FindLargeFilesForm.GetTestAccessor().GenerateCommand([]));
```

Comment:
❔ workaround
```suggestion
            string original = AppSettings.GitCommand;
            try
            {
                AppSettings.GitCommand = @"C:\Program Files\Git\bin\git.exe";
                await Verifier.Verify(FindLargeFilesForm.GetTestAccessor().GenerateCommand([]));
            }
            finally
            {
                AppSettings.GitCommand = original;
            }
```

Reply:
Yes, it was late last night so I didn't get to that. But this, essentially, is the way to fix the tests.
Thank you